app.rb

```diff
@@ -0,0 +1,127 @@
+require './book'
+require './student'
+require './rental'
+require './teacher'
+
+class App
+  def initialize(parent)
+    @parent = parent
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def list_of_books
+    if @books.empty?
+      puts 'No Book In The List.'
+    else
+      puts 'List of Books available.'
+      @books.each { |book| puts "Title: #{book.title} By #{book.author}" }
+    end
+    @parent.show_menu
+  end
+
+  def list_of_people
+    if @people.empty?
+      puts 'List Of People is empty'
+    else
+      puts 'List of People:'
+      @people.each do |person|
+        puts "NAME: #{person.name}, CLASS: #{person.class}, ID: #{person.id}, AGE: #{person.age}"
+      end
+    end
+    @parent.show_menu
+  end
+
+  def create_person
+    puts 'Do you want to create a Student(1) or Teacher(2)? [ENTER NUMBER]:'
+    person_input = gets.chomp
+    case person_input
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Please add a valid input!'
+    end
+    @parent.show_menu
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+    @people.push(Student.new(age, name, parent_permission: parent_permission))
+    puts
+    puts 'Student is successfully created'
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @people.push(Teacher.new(age, specialization, name))
+    puts
+    puts 'Teacher is successfully created'
+  end
+
```

Comment:
- [ ] Good job for fixing the error in teacher creation 👍 However, in place of `age` teacher's `specialization` is being displayed. Kindly fix this to avoid any confusion ⛏

![image](https://github.com/Booze33/OOP-school-library/assets/106119105/6bb1570f-99a5-450a-a461-4a22e43175c0)

- [OPTIONAL] I think it would be better to display the information as mentioned in the [video]() for better readability🧾 💪 What do you think about it? 😃

![image](https://github.com/Booze33/OOP-school-library/assets/106119105/0c66dce7-547c-4edb-a975-14a72f122ece)